src/appmixer/pipedrive/crm/FindPerson/FindPerson.js

```diff
@@ -1,36 +1,103 @@
+                { label: 'Name', value: 'name' },
+                { label: 'Phones', value: 'phones', schema: { type: 'array', items: [] } },
+                { label: 'Emails', value: 'emails', schema: { type: 'array', items: [] } },
+                { label: 'Primary Email', value: 'primary_email' },
+                { label: 'Visible To', value: 'visible_to' },
+                { label: 'Owner', value: 'owner', schema: { type: 'object', properties: { id: { type: 'number', title: 'Owner ID' } } } },
+                { label: 'Organization', value: 'organization' },
+                { label: 'Custom Fields', value: 'custom_fields', schema: { type: 'array', items: [] } },
+                { label: 'Notes', value: 'notes', schema: { type: 'array', items: [] } },
+                { label: 'Update Time', value: 'update_time' }
+            ], outputPortName);
+        } else if (outputType === 'array') {
+            return context.sendJson([
+                {
+                    label: 'Persons',
+                    value: 'records',
+                    schema: {
+                        type: 'array',
+                        items: {
+                            type: 'object',
+                            properties: {
+                                id: { label: 'Person ID', value: 'id' },
+                                type: { label: 'Type', value: 'type' },
+                                name: { label: 'Name', value: 'name' },
+                                phones: { label: 'Phones', value: 'phones', schema: { type: 'array', items: [] } },
+                                emails: { label: 'Emails', value: 'emails', schema: { type: 'array', items: [] } },
+                                primary_email: { label: 'Primary Email', value: 'primary_email' },
+                                visible_to: { label: 'Visible To', value: 'visible_to' },
+                                owner: { label: 'Owner', value: 'owner', schema: { type: 'object', properties: { id: { type: 'number', title: 'Owner ID' } } } },
+                                organization: { label: 'Organization', value: 'organization' },
+                                custom_fields: { label: 'Custom Fields', value: 'custom_fields', schema: { type: 'array', items: [] } },
+                                notes: { label: 'Notes', value: 'notes', schema: { type: 'array', items: [] } },
+                                update_time: { label: 'Update Time', value: 'update_time' }
+                            }
+                        }
                     }
                 }
-            });
+            ], outputPortName);
+        } else if (outputType === 'file') {
+            return context.sendJson([
+                { label: 'File ID', value: 'fileId', schema: { type: 'string', format: 'appmixer-file-id' } }
+            ], outputPortName);
+        } else {
+            // Default to array output
+            return context.sendJson([], outputPortName);
```

Comment:
empty array doesn't mean it's the "Default to array output", does it?  the only possible outputTypes are `file`, `object`, `array` and `first`, error otherwise. What do you think?